backend/tests/test_db/test_toc_transactions.py

```
@@ -328,3 +328,129 @@ async def _fake_update_one(*args, **kwargs):
+        await tx.update_chapter_statuses_with_version_guard(
+            book_id=str(ObjectId()), chapter_ids=["c1"], new_status="in-progress",
+            user_auth_id="someone", expected_version=1,
+        )
+
+
+@pytest.mark.asyncio
+async def test_bulk_status_helper_wrong_owner_not_authorized(seed_book):
+    toc = _toc(version=1, chapters=[{"id": "c1", "title": "C1", "status": "draft"}])
+    book_id, _ = await seed_book(toc=toc, owner_id="real-owner")
+
+    with pytest.raises(ValueError, match="[Nn]ot authorized"):
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)